test/cypress/features/transaction.feature

```diff
@@ -0,0 +1,39 @@
+Feature: Transactions 
+
+  Background:
+    Given I login as testnet_guy on testnet
+    And I wait 5 seconds
+    Given I am on Transactions page
+
+  Scenario: Load Latest Transactions
+    Then I should see 20 transactions in table
+    When I click on showMoreButton
+    Then I should see more than 20 transactions in table
```

Comment:
if devnet is used, clicking more would render a definite value of 40 transactions

Reply:
Fixed